lib/mrtrix3/run.py

```diff
@@ -145,6 +153,14 @@ def set_scratch_dir(self, path):
     self.env['MRTRIX_TMPFILE_DIR'] = path
     self._scratch_dir = path
 
+  # Controls verbosity of invoked MRtrix3 commands, as well as whether or not the
+  #   stderr outputs of invoked commands are propagated to the terminal instantly or
+  #   instead written to a temporary file for read on completion
+  def set_verbosity(self, verbosity):
+    assert isinstance(verbosity, int)
+    self.verbosity = verbosity
+    self.env['MRTRIX_LOGLEVEL'] = str(max(1, verbosity-1))
```

Comment:
This achieves the same as deleted lines 333-335 below: if script is executed with log level 3, i.e. debug mode, then any *MRtrix3* commands invoked will be executed with log level 2, i.e. info mode.